From 3c07505911bae837cd971f54f61287f02fd3c422 Mon Sep 17 00:00:00 2001
From: Christoph Hellmann Santos <christoph.hellmann.santos@ipa.fraunhofer.de>
Date: Thu, 22 Jun 2023 11:17:28 +0200
Subject: [PATCH] Fix dcf-tools

---
 python/dcf-tools/Makefile.am | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/python/dcf-tools/Makefile.am b/python/dcf-tools/Makefile.am
index 9852c84a..31f3942a 100644
--- a/python/dcf-tools/Makefile.am
+++ b/python/dcf-tools/Makefile.am
@@ -23,23 +23,17 @@ EXTRA_DIST += setup.py
 build_base = $(realpath $(builddir))/build
 dist_dir = $(realpath $(builddir))/dist
 
-all-local: python-sdist python-bdist_wheel
+all-local: python-sdist
 
 clean-local:
 	rm -rf $(build_base) $(dist_dir) $(srcdir)/*.egg-info $(builddir)/*.egg-info
 
 install-exec-local: python-install
 
-.PHONY: python-bdist_wheel
-python-bdist_wheel:
-if HAVE_PYTHON3
-	@$(PYTHON3_ENV) $(PYTHON3) $(srcdir)/setup.py \
-		bdist_wheel --dist-dir $(dist_dir)
-endif
-
 .PHONY: python-install
 python-install:
 if HAVE_PYTHON3
+	@echo $($(DESTDIR)$(prefix))
 	@$(PYTHON3_ENV) $(PYTHON3) $(srcdir)/setup.py \
 		install --prefix $(DESTDIR)$(prefix) --root /
 endif
-- 
2.34.1

