From d8aee7e541a264a52f3831248835cc618f41db95 Mon Sep 17 00:00:00 2001
From: Christoph Hellmann Santos <christoph.hellmann.santos@ipa.fraunhofer.de>
Date: Thu, 22 Jun 2023 12:00:14 +0200
Subject: [PATCH] Fix dcf-tools

---
 python/dcf-tools/Makefile.am | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/python/dcf-tools/Makefile.am b/python/dcf-tools/Makefile.am
index 9852c84a..d7915fe1 100644
--- a/python/dcf-tools/Makefile.am
+++ b/python/dcf-tools/Makefile.am
@@ -23,30 +23,15 @@ EXTRA_DIST += setup.py
 build_base = $(realpath $(builddir))/build
 dist_dir = $(realpath $(builddir))/dist
 
-all-local: python-sdist python-bdist_wheel
-
 clean-local:
 	rm -rf $(build_base) $(dist_dir) $(srcdir)/*.egg-info $(builddir)/*.egg-info
 
 install-exec-local: python-install
 
-.PHONY: python-bdist_wheel
-python-bdist_wheel:
-if HAVE_PYTHON3
-	@$(PYTHON3_ENV) $(PYTHON3) $(srcdir)/setup.py \
-		bdist_wheel --dist-dir $(dist_dir)
-endif
-
 .PHONY: python-install
 python-install:
 if HAVE_PYTHON3
 	@$(PYTHON3_ENV) $(PYTHON3) $(srcdir)/setup.py \
-		install --prefix $(DESTDIR)$(prefix) --root /
+		install --prefix $(DESTDIR)$(prefix) --install-scripts $(DESTDIR)$(prefix)/bin
 endif
 
-.PHONY: python-sdist
-python-sdist:
-if HAVE_PYTHON3
-	@cd $(srcdir); $(PYTHON3_ENV) $(PYTHON3) setup.py \
-		sdist --dist-dir $(dist_dir)
-endif
-- 
2.34.1

